ml-completion/src/main/kotlin/org/rust/ml/RustMLRankingProvider.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ml
+
+import com.intellij.internal.ml.catboost.CatBoostJarCompletionModelProvider
+import com.intellij.lang.Language
+import org.rust.lang.RsLanguage
+
+@Suppress("UnstableApiUsage")
+class RustMLRankingProvider : CatBoostJarCompletionModelProvider("Rust", "rust_features", "rust_model") {
```

Comment:
I've chosen the name with "Rust" word because all the `CatBoostJarCompletionModelProvider` subclasses use the full language names instead of the abbreviations (including our experimental model provider for EAP builds `ExperimentRustMLRankingProvider`). So it would be easier to search across all the implementations with `Find external Usages` action for those who unfamiliar with our "Rs" prefix, for example